packages/fluttium_cli/lib/src/commands/test_command/reporters/compact_reporter.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:fluttium_cli/src/commands/test_command/reporters/reporters.dart';
+import 'package:fluttium_driver/fluttium_driver.dart';
+import 'package:mason/mason.dart';
+
+class CompactReporter extends Reporter {
+  CompactReporter(super.driver, {required super.watch, required this.logger}) {
+    if (!watch) return;
+    throw UnsupportedError('The compact reporter does not support watch mode.');
```

Comment:
nit but it would've been a bit clearer to adjust this like:

```dart
if (watch) {
  throw UnsupportedError('The compact reporter does not support watch mode.');
}
```